### PR TITLE
Fix crash in DocViewer

### PR DESCRIPTION
**Pull request by armstrongnate**
refs: MBL-14723
affects: student, teacher
release note: Fixed a crash when adding annotations

test plan:
- Add some annotations to a document
- Add some comments to an annotation
- Should not crash

### CODE REVIEW

File: Core/Core/DocViewer/DocViewerAnnotationProvider.swift [hunk 1]
```diff
@@ -110,7 +110,7 @@ class DocViewerAnnotationProvider: PDFContainerAnnotationProvider {
             } else {
                 self?.docViewerDelegate?.annotationDidFailToSave(error: error ?? APIDocViewerError.noData)
             }
-        }
+        } }
     }
 
     private func delete(_ id: String) {
```
Comment (cobbal) [inline Core/Core/DocViewer/DocViewerAnnotationProvider.swift:116]: `delete` should probably be fixed too.

**Review by cobbal — CHANGES_REQUESTED** [review]

File: Core/Core/DocViewer/DocViewerAnnotationProvider.swift (hunk 1 above)
Comment (armstrongnate) [reply Core/Core/DocViewer/DocViewerAnnotationProvider.swift:116]: great call, thanks.

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49145423/p/07290dc90a038bf60d45bed3e5e4f435">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49145449/p/bb9f8911150557403cfaea01e51bf655">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/49145393/p/a0d30b3e5eb2af10909fd7c27fadf1de">
    
  </a>
</details>

**Comment by inst-danger** [comment]
Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.07% | 93.17% | -0.1%

[MBL-14723](https://instructure.atlassian.net/browse/MBL-14723)

#### Affected Apps: Student, Teacher

#### Release Note: 
Fixed a crash when adding annotations
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Review by cobbal — APPROVED** [review]

**Pull request merged**